docker/dockerfiles/Dockerfile_celery

```diff
@@ -0,0 +1,32 @@
+FROM dimagi/commcarehq_base:latest
+
+RUN apt-get update
+RUN apt-get install -y netcat
```

Comment:
I don't think it is - but I support putting it in the base image. I just added it to make it possible to wait for the web container.